src/index.html

```diff
@@ -48,14 +48,15 @@
 
     <div class="content-wrapper">
       <div class="title-container">
-        <h1 class="app-title">Splitter</h1>
+        <h1 class="app-title">Spli</h1>
+        <h1 class="app-title">tter</h1>
```

Comment:
just a note - I'd like this to be one `<h1>` with styling logic to split it, but this will do for now.